src/renderer/components/ft-slider/ft-slider.css

```diff
@@ -1,250 +1,257 @@
 }
 
 .pure-material-slider > input:disabled::-webkit-slider-thumb {
-    background-color: rgb(var(--pure-material-onsurface-rgb, 0, 0, 0));
-    color: rgb(var(--pure-material-surface-rgb, 255, 255, 255)); /* Safari */
-    box-shadow: 0 0 0 1px rgb(var(--pure-material-surface-rgb, 255, 255, 255)) !important;
-    transform: scale(4, 4);
+  background-color: rgb(var(--pure-material-onsurface-rgb, 0, 0, 0));
+  color: rgb(var(--pure-material-surface-rgb, 255, 255, 255)); /* Safari */
+  box-shadow: 0 0 0 1px rgb(var(--pure-material-surface-rgb, 255, 255, 255)) !important;
+  transform: scale(4, 4);
 }
 
 /* Moz | Track */
 .pure-material-slider > input::-moz-range-track {
-    margin: 17px 0;
-    border-radius: 1px;
-    width: 100%;
-    height: 2px;
-    background-color: var(--accent-color-opacity4);
+  margin-block: 0;
+  margin-auto: 17px;
+  border-radius: 1px;
+  width: 100%;
+  height: 2px;
+  background-color: var(--accent-color-opacity4);
 }
 
 /* Moz | Thumb */
 .pure-material-slider > input::-moz-range-thumb {
-    appearance: none;
-    -moz-appearance: none;
-    border: none;
-    border-radius: 50%;
-    height: 2px;
-    width: 2px;
-    background-color: var(--accent-color);
-    transform: scale(6, 6);
-    transition: box-shadow 0.2s;
+  appearance: none;
+  -moz-appearance: none;
+  border: none;
+  border-radius: 50%;
+  height: 2px;
+  width: 2px;
+  background-color: var(--accent-color);
+  transform: scale(6, 6);
+  transition: box-shadow 0.2s;
 }
 
 /* Moz | Progress */
 .pure-material-slider > input::-moz-range-progress {
-    border-radius: 1px;
-    height: 2px;
-    background-color: var(--accent-color);
+  border-radius: 1px;
+  height: 2px;
+  background-color: var(--accent-color);
 }
 
 /* Moz | Hover, Focus */
 .pure-material-slider:hover > input:hover::-moz-range-thumb {
-    box-shadow: 0 0 0 2px var(--accent-color-opacity1);
+  box-shadow: 0 0 0 2px var(--accent-color-opacity1);
 }
 
 .pure-material-slider > input:focus::-moz-range-thumb {
-    box-shadow: 0 0 0 2px var(--accent-color-opacity2);
+  box-shadow: 0 0 0 2px var(--accent-color-opacity2);
 }
 
 .pure-material-slider:hover > input:focus::-moz-range-thumb {
-    box-shadow: 0 0 0 2px var(--accent-color-opacity3);
+  box-shadow: 0 0 0 2px var(--accent-color-opacity3);
 }
 
 /* Moz | Active */
 .pure-material-slider > input:active::-moz-range-thumb {
-    box-shadow: 0 0 0 2px var(--accent-color-opacity4) !important;
+  box-shadow: 0 0 0 2px var(--accent-color-opacity4) !important;
 }
 
 /* Moz | Disabled */
 .pure-material-slider > input:disabled::-moz-range-track {
-    background-color: rgba(var(--pure-material-onsurface-rgb, 0, 0, 0), 0.38);
+  background-color: rgba(var(--pure-material-onsurface-rgb, 0, 0, 0), 0.38);
 }
 
 .pure-material-slider > input:disabled::-moz-range-progress {
-    background-color: rgba(var(--pure-material-onsurface-rgb, 0, 0, 0), 0.87);
+  background-color: rgba(var(--pure-material-onsurface-rgb, 0, 0, 0), 0.87);
 }
-
+O
```

Comment:
What kind of secret code is this? :)
```suggestion
```

Reply:
This was a secret code that added custom playlists, but I'll remove it to keep us modest.